presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/MongoServer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mongodb;
+
+import com.google.common.net.HostAndPort;
+import org.testcontainers.containers.GenericContainer;
+
+import java.io.Closeable;
+
+public class MongoServer
+        implements Closeable
+{
+    private static final int MONGO_PORT = 27017;
+
+    private final GenericContainer<?> dockerContainer;
+
+    public MongoServer()
+    {
+        this.dockerContainer = new GenericContainer<>("mongo:3.2.0")
```

Comment:
I missed that before. This uses 3.2.0 whereas previously we had `<mongo-server.version>1.5.0</mongo-server.version>`. `mongo:1.5.0` doesn't exist, so let's call it out in the commit message. Maybe:

```
Use dockerized mongo and upgrade mongo version in tests
```

Reply:
The `1.5.0` doesn't mean mongodb version. It is the test library's (`de.bwaldvogel/mongo-java-server`) release version. It seems 1.5.0 equals to mongodb `3.0.0`. 
https://github.com/bwaldvogel/mongo-java-server/blob/v1.5.0/core/src/main/java/de/bwaldvogel/mongo/backend/AbstractMongoBackend.java#L40

Let me downgrade just in case. I will upgrade the version in https://github.com/prestosql/presto/pull/2156, it requires `3.4.0`.

Reply:
For some reason, 3.0.0 docker image is very unstable. The test failed several times on both GitHub Actions and my local environment. Let me update to 3.4.0 with suggested commit message in this PR. I confirmed 3.2.0 and 3.4.0 are stable.